openedx/core/djangoapps/schedules/resolvers.py

```diff
@@ -357,20 +357,21 @@ def schedules_for_bin(self):
                         user, week_num, enrollment.course_id
                     )
                 )
+                # continue to the next schedule, don't yield an email for this one
```

Comment:
Nit: Instead of using an `else` here, change this to a `continue` and then you can remove the comment.

Reply:
Ha. @Rabia23 suggested I use `try-except-else` here instead of `continue`. I don't really care which one we use, but I don't know which I should go with now.

Reply:
Oh, I didn't realize.  I think `continue` in this case is the least error-prone for the future.  2 reasons:
1. Someone doesn't accidentally add additional code after the `else` within the `for` loop.
2. Less likely the comment would get outdated if the comment is no longer needed.